documentation/src/main/asciidoc/topics/proc_cli_creating_users.adoc

```diff
@@ -1,47 +1,131 @@
-[id='user_create-{context}']
+[id='creating-users_{context}']
 = Creating and Modifying Users
-{brandname} Server requires users to authenticate against a default property realm.
-Before you can access {brandname} Server, you must add credentials by creating at least one user and a password.
-You can also add and modify the security authorization groups to which users belong.
+Add {brandname} user credentials and assign permissions to control access to data.
 
-.Procedure
+{brandname} server installations use a property realm to authenticate users for the Hot Rod and REST endpoints.
+This means you need to create at least one user before you can access {brandname}.
 
-. Open a terminal in `{server_home}`.
-. Create and modify {brandname} users with the [command]`user` command.
+By default, users also need roles with permissions to access caches and interact with {brandname} resources.
+You can assign roles to users individually or add users to groups that have role permissions.
+
+You create users and assign roles with the [command]`user` command in the {brandname} command line interface (CLI).
 
 [TIP]
 ====
-Run [command]`help user` for more details about using the command.
+Run [command]`help user` from a CLI session to get complete command details.
 ====
 
-.Creating users and passwords
+== Adding Credentials
+
+You need an `admin` user for the {brandname} Console and full control over your {brandname} environment.
+For this reason you should create a user with `admin` permissions the first time you add credentials.
 
-* Linux
+.Procedure
+
+. Open a terminal in `{server_home}`.
+. Create an `admin` user with the [command]`user create` command in the CLI.
 +
-[source,options="nowrap",subs=attributes+]
+[source,sh,options="nowrap",subs=attributes+]
 ----
-include::cmd_examples/user_tool_nix.adoc[]
+$ bin/cli.sh user create myuser -p changeme -g admin
+----
++
+Alternatively, the username "admin" automatically gets `admin` permissions.
```

Comment:
Maybe we should state that this is because the cluster-role-mapper (similarly to the identity-role-mapper) will assign roles to a user when one of that user's principal names match one of the role names.
A user named 'admin' will have its name principal match.
A user belonging to the group `admin` will have its group principal match.

Reply:
I put the info about the cluster-role-mapper as an important note in the default users and permissions topic: https://github.com/infinispan/infinispan/pull/9163/files#diff-4bd6db5f2438323596192eacd5a3aac4e5423f65cd8f2c1883f43d8c5f5d6effR7 

In this procedure I thought it might be preferable to keep phrasing more like a "shortcut" or fast path. My thinking is that users will encounter this procedure in the Getting Started sections so will probably just want to create an admin user for the first time to get in. Then they can move on to understanding roles and permissions more deeply.

If you do think we need to rephrase this line, I can work with that. But that detail about the cluster-role-mapper is in docs.